common/shared.c

```diff
@@ -336,6 +336,10 @@ void ft_parsecsopts(int op, char *optarg, struct cs_opts *opts)
 		break;
 	case 'p':
 		opts->port = optarg;
+		opts->src_port = optarg;
```

Comment:
Yes, I'll be removing the port when I update other examples.

Sounds good. -p for dst_port and -b for src_port.